Dockerfile

```diff
@@ -3,11 +3,14 @@ MAINTAINER Remi Hakim @remh
 RUN rm /bin/sh && ln -s /bin/bash /bin/sh
 
 RUN apt-get update && apt-get install -y \
-    git \
     curl \
     procps \
     fakeroot
 
+RUN echo "deb http://http.debian.net/debian squeeze-backports main" >/etc/apt/sources.list.d/squeeze-backports.list
```

Comment:
I could've done that, but I didn't want to change the versions you were using for `curl`, `procps` and `fakeroot` - so I did it in a separate step.